ci-operator/config/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-master.yaml

```diff
@@ -63,11 +63,10 @@ tests:
   openshift_installer_custom_test_image:
     cluster_profile: aws
     from: stable:cluster-api-actuator-pkg
-- as: e2e-aws-upgrade
-  commands: TEST_SUITE=all run-upgrade-tests
-  openshift_installer:
+- as: e2e-upgrade
```

Comment:
Is there a reason to remove the 'aws' part of this job name?  With aws in the name, it makes it more clear on PRs what is running where.  This repo should never, for instance, run an e2e job on GCP.

Reply:
Makes sense, reverted the rename